tests/acceptance/tests/Settings/WishlistGuestFunctionalities.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { test } from '@fixtures/AcceptanceTest';
+
+test('Guest Customer is able to add and remove products to the wishlist',{ tag: '@Wishlist' }, async ({ 
+    TestDataService,
+    ShopCustomer,
+    StorefrontHome,
+    AddProductToWishlist,
+    StorefrontWishlist,
+    AddProductToCartFromWishlist,
+    Login,
+    StorefrontOffCanvasCart,
+}) => {
+    await TestDataService.setSystemConfig({ 'core.cart.wishlistEnabled': true });
+    const product1 = await TestDataService.createBasicProduct();
+    const product2 = await TestDataService.createBasicProduct();
+    const product1Locators = await StorefrontHome.getListingItemByProductId(product1.id);
+    const product2Locators = await StorefrontHome.getListingItemByProductId(product2.id);
+
+    await test.step('Accept all cookies and reload page', async () => {
+        await TestDataService.setSystemConfig({ 'core.basicInformation.acceptAllCookies': true });
+        await ShopCustomer.goesTo(StorefrontHome.url());
+        await StorefrontHome.consentAcceptAllCookiesButton.click();
+        await StorefrontHome.page.reload();
+    });
+
+    await test.step('Add product1 to the wishlist and verify wishlist count updates to 1', async () => {
+        await ShopCustomer.attemptsTo(AddProductToWishlist(product1));
+        await ShopCustomer.expects(product1Locators.wishlistAddedIcon).toBeVisible();
+        await ShopCustomer.expects(StorefrontHome.wishlistBasket).toHaveText('1');
+    });
+
+    await test.step('Add product1 to the cart from wishlist and verify cart total is same with product price', async () => {
+        await ShopCustomer.attemptsTo(AddProductToCartFromWishlist(product1));
+        const productPrice = await product1Locators.productPrice.innerText();
+        const offCanvasSubtotal = await StorefrontOffCanvasCart.subTotalPrice.innerText();
+        ShopCustomer.expects(offCanvasSubtotal).toBe(productPrice);
```

Comment:
You could use product price from test data service. 
```suggestion
        ShopCustomer.expects(offCanvasSubtotal).toBe(product1.price);
```

Reply:
Here, the value is returned as an array, not in the same format as the UI, as shown in the attached example. If there's no issue with this, we can keep it as it is.

Error: expect(received).toBe(expected) // Object.is equality

Expected: `[{"apiAlias": "price", "currencyId": "b7d2554b0ce847cd82f3ac9bd1c0dfca", "extensions": [], "gross": 10, "linked": false, "listPrice": null, "net": 8.4, "percentage": null, "regulationPrice": null}]`
Received: "€10.00"

Reply:
@yusufttur `product1.price` is an Array Object of prices see [here](https://github.com/shopware/acceptance-test-suite/blob/2a06c23818afb204b2b369c6a12e6b47dbbf3256/src/services/TestDataService.ts#L1876) , therefore you see that Object syntax as Orkun was provided.

A better solution to avoid a bit the "`.innerText`" syntax you could use:

```
 const productPrice = await product1Locators.productPrice.innerText(); //need string to compare
 await ShopCustomer.expects(StorefrontOffCanvasCart.subTotalPrice).toContainText(productPrice)
```